src/app/components/harvest/screens/metadata-review/metadata-review.component.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { MockBawApiModule } from "@baw-api/baw-apiMock.module";
+import { ConfirmationComponent } from "@components/harvest/components/modal/confirmation.component";
+import { HarvestStagesService } from "@components/harvest/services/harvest-stages.service";
+import {
+  Harvest,
+  HarvestStatus
+} from "@models/Harvest";
+import { Project } from "@models/Project";
+import { NgbModal } from "@ng-bootstrap/ng-bootstrap";
+import {
+  createRoutingFactory,
+  SpectatorRouting,
+  SpyObject,
+} from "@ngneat/spectator";
+import { SharedModule } from "@shared/shared.module";
+import { generateHarvest } from "@test/fakes/Harvest";
+import {
+  generateProject,
+  generateProjectMeta
+} from "@test/fakes/Project";
+import { MockProvider } from "ng-mocks";
+import { ToastrService } from "ngx-toastr";
+import { MetadataReviewComponent } from "./metadata-review.component";
+
+describe("MetaDatReviewComponent", () => {
+  let spec: SpectatorRouting<MetadataReviewComponent>;
+  let modalService: NgbModal;
+  let stages: SpyObject<HarvestStagesService>;
+  let defaultProject: Project;
+  let defaultHarvest: Harvest;
+
+  const createComponent = createRoutingFactory({
+    declarations: [ConfirmationComponent],
+    component: MetadataReviewComponent,
+    providers: [
+      MockProvider(HarvestStagesService, {
+        project: defaultProject,
+        harvest: defaultHarvest,
+        transition: (_stage: HarvestStatus) => {}
+      }),
+    ],
+    imports: [MockBawApiModule, SharedModule],
+    mocks: [ToastrService],
+  });
+
+  function setup() {
+    spec = createComponent({ detectChanges: false });
+
+    modalService = spec.inject(NgbModal);
+
+    stages = spec.inject<SpyObject<HarvestStagesService>>( HarvestStagesService as any );
+    stages.transition = jasmine.createSpy("transition") as any;
```

Comment:
is there a reason this setup is different from the list spec?

Reply:
This spec was created before the list spec

The spy's on the list spec are not working properly, so I'm considering lifting this setup from metadata-review to the list spec since I know that the setup on metadata-review as expected. This would also work well for consistency, as the batch-upload component uses a similar setup as the metadata-review component spec